### PR TITLE
first version

### PR DESCRIPTION
**Pull request by Ibrahim-Jarada**

### CODE REVIEW

**Review by RandSohail — APPROVED** [review]

**Review by RandSohail — APPROVED** [review]

File: src/controllers/admin/getDashboardSettings.ts [hunk 1]
```diff
@@ -1,16 +1,19 @@
+import { IDashboardSettings } from 'db-models-nc';
 import { Request, Response, NextFunction } from 'express';
-import { getDashboardSettings } from '../../services';
-import { constants } from '../../helpers';
+// import { getDashboardSettings } from '../../services';
```
Comment (zaher-aa) [inline src/controllers/admin/getDashboardSettings.ts:3]: comment

File: src/controllers/admin/getDashboardSettings.ts [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 
 export default async (_request: Request, response: Response, next: NextFunction): Promise<void> => {
   try {
-    const data = await getDashboardSettings();
-    const { regularVariables, encryptedVariables } = data;
-    const dashboardSettings = { ...regularVariables, ...encryptedVariables };
+    // const data2 = await getDashboardSettings();
+    // const { regularVariables, encryptedVariables } = data2;
+    // const data = { ...regularVariables, ...encryptedVariables };
```
Comment (zaher-aa) [inline src/controllers/admin/getDashboardSettings.ts:9]: comment

File: src/controllers/admin/getDashboardSettings.ts [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
+    const data = _request.app.get('settings') as IDashboardSettings;
+
+    console.log(data);
```
Comment (zaher-aa) [inline src/controllers/admin/getDashboardSettings.ts:12]: remove clg

File: src/controllers/admin/getDashboardSettings.ts [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
 
 export default async (_request: Request, response: Response, next: NextFunction): Promise<void> => {
   try {
-    const data = await getDashboardSettings();
-    const { regularVariables, encryptedVariables } = data;
-    const dashboardSettings = { ...regularVariables, ...encryptedVariables };
+    // const data2 = await getDashboardSettings();
+    // const { regularVariables, encryptedVariables } = data2;
+    // const data = { ...regularVariables, ...encryptedVariables };
+    const data = _request.app.get('settings') as IDashboardSettings;
```
Comment (zaher-aa) [inline src/controllers/admin/getDashboardSettings.ts:10]:
```suggestion
    const data = request.app.get('settings') as IDashboardSettings;
```

File: src/controllers/users/getUserDataByID.ts [hunk 5]
```diff
@@ -4,13 +4,13 @@ import { constants, errorMessages, dto } from '../../helpers';
...
 : Promise<void> => {
-  const { httpStatus, messages } = constants;
+  const { httpStatus } = constants;
   const { userId } = dto.generalDTO.userIdDTO(request);
   try {
     const user = await getUserById(userId);
     if (!user) throw errorMessages.NOT_EXIST_ERROR;
 
-    response.status(httpStatus.OK).json({ message: messages.authResponse.SUCCESS, data: user });
+    response.status(httpStatus.OK).json({ data: user });
```
Comment (zaher-aa) [inline src/controllers/users/getUserDataByID.ts:13]: no necessary `httpStatus.OK`

File: src/middleware/error/index.ts [hunk 6]
```diff
@@ -13,6 +13,8 @@ interface IError extends Error {
 
 const serverError = (error: IError, _request: Request, response: Response, _next: NextFunction)
 :void => {
+  console.log(error);
+
```
Comment (zaher-aa) [inline src/middleware/error/index.ts:17]: remove clg

**Review by zaher-aa — APPROVED** [review]

**Pull request merged**